### PR TITLE
change packagecheck name

### PR DESCRIPTION
**Pull request by sahil-lakhwani**
Signed-off-by: sahil-lakhwani <sahilakhwani@gmail.com>

### CODE REVIEW

**Review by nikhilbandi — APPROVED** [review]
Looks good.

File: pkg/platform/centos/centos.go [hunk 1]
```diff
@@ -29,7 +29,7 @@ func (c *CentOS) Check() []platform.Check {
 	checks = append(checks, platform.Check{"PyCliCheck", result, err})
```
Comment (anupbarve) [inline pkg/platform/centos/centos.go:29]: Can we rename this as well. PyCLICheck probably doesn't make sense to the user.

**Review by anupbarve — CHANGES_REQUESTED** [review]

File: pkg/platform/centos/centos.go (hunk 1 above)
Comment (sahil-lakhwani) [reply pkg/platform/centos/centos.go:29]: Does "Python CLI Removal" sound good?

Comment (anupbarve) [reply pkg/platform/centos/centos.go:29]: yeah, sounds good.

**Review by anupbarve — APPROVED** [review]

**Pull request merged**